device/alpaka/src/seeding/spacepoint_binning.cpp

```diff
@@ -90,11 +90,8 @@ traccc::details::spacepoint_grid_types::buffer spacepoint_binning::operator()(
         grid_capacities_buff;
 
     // Now define the Alpaka Work division
-    auto const deviceProperties = ::alpaka::getAccDevProps<Acc>(devAcc);
-    auto const maxThreadsPerBlock = deviceProperties.m_blockThreadExtentMax[0];
-    auto const threadsPerBlock = maxThreadsPerBlock;
-    auto const blocksPerGrid =
-        (sp_size + threadsPerBlock - 1) / threadsPerBlock;
+    const Idx threadsPerBlock = getWarpSize<Acc>() * 8;
+    const Idx blocksPerGrid = (sp_size + threadsPerBlock - 1) / threadsPerBlock;
```

Comment:
Is there a reason why you are not performing the check to see if the block size exceeds the device maximum?

Reply:
It mirrors the lack of checking in the CUDA version (actually, CUDA doesn't check the threads either I don't think?)....which isn't really a strong argument, more just that I mirrored the style there.

Perhaps a more repo wide discussion we need to have?

It seems right now we assume we are so far under the limits, it doesn't really matter....which is reasonable until it isn't. I've certainly hit some issues when running in FP64 mode, as some of the thread variables don't fit when using `double` vs `float` etc.

Reply:
Well, the reason for this is that the limit in CUDA is very clearly defined at 1024. I don't know how this limit is set in Alpaka, especially for e.g. CPU devices. Do you know more about this?

Reply:
I didn't find anything about the limit in Alpaka documentation so far

Reply:
Okay, I'm sure we can figure this out later. Let's move forward with this PR.